siac/mangencmd.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+var (
+	mangenCmd = &cobra.Command{
+		Use:   "man-generation [path]",
+		Short: "Creates unix style manpages.",
+		Long: "Creates a man pages at the specified " +
+			"directory.\n\n",
+
+		Run: wrap(mangencmd),
```

Comment:
no extra newline necessary. Also, `Long` comments should use ` rather than ". Follow the style of other commands in the same package.